94-appendixD.Rmd

```diff
@@ -1627,3 +1627,29 @@ US_births_1994_2003 %>%
 ```
 
 The dates with the fewest number of births in the US was 12/25 of the years of 2001, 2000, 2003, 2002, and 1999. Because it is Christmas Day and hospitals don't generally induce labor on that day. 
+
+
+
+***
+
+
+
+## Appendix A2 Solutions
+
+```{r, include=FALSE, purl=FALSE}
+chap <- "A2"
+lc <- 0
+```
+
+
+**`r paste0("(LC", chap, ".", (lc <- lc + 1), ")")`** What proportion of the area under the normal curve is less than 3? Greater than 12? Between 0 and 12?
+
+* Less than 3: 3 is one standard deviation less than the mean of 6, since $\frac{3 - \mu}{\sigma} = \frac{3-6}{3} = -1$. Thus we compute the area to the left of z = -1 in Figure \@ref(fig:normal-rule-of-thumb): 0.15% + 2.35% + 13.5% = 16%.
+* Greater than 12: 12 is two standard deviations greater than the mean of 6, since $\frac{12 - \mu}{\sigma} = \frac{12-6}{3} = +2$, we compute the area to the right of z = +2 in Figure \@ref(fig:normal-rule-of-thumb): 2.35% + 0.15% = 2.5%.
+* Between 0 and 12: 0 is two standard deviations less than the mean of 6, since $\frac{0 - \mu}{\sigma} = \frac{0-6}{3} = -2$. Thus we compute the area between z = -2 and z = +2 in Figure \@ref(fig:normal-rule-of-thumb): 13.5% + 34% + 34% + 13.5% = 95%.
+
+**`r paste0("(LC", chap, ".", (lc <- lc + 1), ")")`** What is the 2.5th percentile of the area under the normal curve? The 97.5th percentile? The 100th percentile?
+
+* 2.5th percentile: Starting from the left of Figure \@ref(fig:normal-rule-of-thumb), since 0.15% + 2.35% = 2.5%, the 2.5th percentile is z = - 2. However, this is in standard units. Thus we need the value in the normal distribution that is two standard deviations lower than the mean: $\mu - 2 \cdot\sigma = 6 - 2 \cdot 3 = 0$.
+* 95th percentile: Starting from the left of Figure \@ref(fig:normal-rule-of-thumb), since 0.15% + 2.35% + 13.5% + 34% + 34% + 13.5% = 97.5%, the 97.5th percentile is z = +2. However, this is in standard units. Thus we need the value in the normal distribution that is two standard deviations higher than the mean: $\mu + 2 \cdot\sigma = 6 + 2 \cdot 3 = 12$.
```

Comment:
```suggestion
* 97.5th percentile: Starting from the left of Figure \@ref(fig:normal-rule-of-thumb), since 0.15% + 2.35% + 13.5% + 34% + 34% + 13.5% = 97.5%, the 97.5th percentile is z = +2. However, this is in standard units. Thus we need the value in the normal distribution that is two standard deviations higher than the mean: $\mu + 2 \cdot\sigma = 6 + 2 \cdot 3 = 12$.
```